config/crd/bases/infrastructure.cluster.x-k8s.io_vcdclustertemplates.yaml

```diff
@@ -289,9 +274,9 @@ spec:
                       org:
                         type: string
                       ovcdZoneConfigMap:
-                        description: |-
-                          OVDCZoneConfigMap defines the name of a config map storing the mapping Zone -> OVDC in a Multi-AZ
-                          deployment. e.g. zone1 -> ovdc1, zone2 -> ovdc2
+                        description: OVDCZoneConfigMap defines the name of a config
```

Comment:
I think this change is because the conversion-gen is not updated. Could you please delete the binary and re-get it using `make conversion` or something similar.